logstash-core/lib/logstash/config/config_ast.rb

```diff
@@ -230,18 +232,18 @@ def compile_initializer
 
     def compile
       case plugin_type
-      when "input"
-        return "start_input(#{variable_name})"
-      when "filter"
-        return <<-CODE
+        when "input"
```

Comment:
Are these cosmetic indentation changes part of _our_ style guidelines?  Or are these general ruby style guidelines?  I've seen the unindented style much more frequently, so I'm curious.